### PR TITLE
Schema for permissions

### PR DESCRIPTION
**Pull request by jmelis**
This MR includes two changes:

- It turns out that draft04 does not support the keyword `const` so it wasn't doing anything. It has been replaced with a single element `enum` list.
- Certain types of permissions require specific associated attributes. These have been modelled into permission-1.yml schema.

### CODE REVIEW

File: schemas/access/permission-1.yml [hunk 1]
```diff
@@ -13,15 +13,54 @@ properties:
...
+    org:
+      type: string
+  required:
+  - org
 required:
 - $schema
 - labels
 - name
 - description
-- service
```
Comment (jakedt) [inline schemas/access/permission-1.yml]: Why did you remove this? It seems like it would help.

File: schemas/access/permission-1.yml [hunk 2]
```diff
@@ -13,15 +13,54 @@ properties:
     type: string
   service:
     type: string
-    enum:
```
Comment (jakedt) [inline schemas/access/permission-1.yml]: The $schema above could be an enum with the name of this file defined, no?

File: schemas/access/permission-1.yml (hunk 1 above)
Comment (jmelis) [reply schemas/access/permission-1.yml]: The service param is implicitly required by the oneOfs. Putting it in the required attribute outside is a bit redundant I think. I verified that it doesn't validate if service is missing

File: schemas/access/permission-1.yml (hunk 2 above)
Comment (jmelis) [reply schemas/access/permission-1.yml]: good idea! will update PR after applying that to all files

**Comment by jakedt** [comment]
I verified on an online validator that I could validate without the service
at all.

File: schemas/access/permission-1.yml (hunk 1 above)
Comment (jmelis) [reply schemas/access/permission-1.yml]: Re-adding the service, for some reason it validates with the current library, but it might be bugged.
